### PR TITLE
Document latex Completion.

### LINKED ISSUE #9913: Document \latex and \unicode completions
cf comment https://github.com/ipython/ipython/pull/6380#issuecomment-242601680

### PR DESCRIPTION
**Pull request by Carreau**
Closes #9913

-- 

cc @willingc ?

### CODE REVIEW

File: IPython/core/completer.py [hunk 1]
```diff
@@ -9,6 +9,60 @@
 for normal classic Python code, as well as completer for IPython specific
 Syntax like magics.
 
+Latex and Unicode completion
+============================
+
+IPython and compatible frontend not only can complete your code, but can help
```
Comment (willingc) [inline IPython/core/completer.py]: frontends

File: IPython/core/completer.py [hunk 2]
```diff
@@ -9,6 +9,60 @@
...
+you to input a wide range of characters. In particular we allow you to insert
+unicode character using the tab completion mechanism.
```
Comment (willingc) [inline IPython/core/completer.py]: a unicode

File: IPython/core/completer.py [hunk 3]
```diff
@@ -9,6 +9,60 @@
...
+
+
+.. code::
+
+    \\greek small letter alpha<tab>
+    α
+
+
+Only valid Python identifiers will complete. Combining characters (like arrow or
+dots) are also available, unlike latex they need to be but after the their
```
Comment (willingc) [inline IPython/core/completer.py]: s/but/put

**Comment by willingc** [comment]
A few small edits and then :shipit:

**Review by srinivasreddy — APPROVED** [review]

**Comment by Carreau** [comment]
Fixed. Thanks !

**Pull request merged**